content/news/XXXX-XX-XX-dvs-internals-pt1.md

```diff
@@ -0,0 +1,156 @@
+title: "How Does Timecode Vinyl Actually Work? (Pt. 1)"
```

Comment:
I think you should remove part 1 here to avoid conflicts.

Reply:
I'm going to rebase once pt. 1 has been merged. This PR is based on the other one, because otherwise I woudn't be able to link to the other post without warnings.